processor/src/test/java/org/mapstruct/ap/testutil/runner/GeneratedSource.java

```diff
@@ -31,6 +31,7 @@
 import org.mapstruct.ap.testutil.assertions.JavaFileAssert;
 
 import static org.assertj.core.api.Assertions.fail;
+import static org.mapstruct.factory.Mappers.getFlatName;
```

Comment:
As I wrote in the `Mapper`, I would write an implementation here and hide the one in `Mappers`